wot-wg-2023-draft.html

```diff
@@ -420,6 +418,10 @@ <h3>
             <!-- Since we plan to publish an update soon the URL might need to change -->
             Draft state: <a href="https://www.w3.org/TR/2020/NOTE-wot-scripting-api-20201124/">Group Note</a>
           </li>
+          <li>Binding Documents for Specific Protocol, Payload and Ecosystems (W3C Notes): These documents will take 
+            the requirements specified in <a href="binding-deliverable">Web of Things (WoT) Binding Templates</a> and 
+            specify how protocols and payload formats should be used to allow integration of IoT systems and ecosystems.
```

Comment:
Technically, as written it means "all" IoT systems and ecosystems.  But each doc only addresses one subclass.
```suggestion
            specify how protocols and payload formats should be used to allow integration of specific classes of IoT systems and ecosystems.
```